lighthouse-service/go.sum

```diff
@@ -181,6 +186,10 @@ github.com/jstemmer/go-junit-report v0.0.0-20190106144839-af01ea7f8024/go.mod h1
 github.com/julienschmidt/httprouter v1.2.0/go.mod h1:SYymIcj16QtmaHHD7aYtjjsJG7VTCxuUUipMqKk8s4w=
 github.com/kelseyhightower/envconfig v1.4.0 h1:Im6hONhd3pLkfDFsbRgu68RDNkGF1r3dvMUtDTo2cv8=
 github.com/kelseyhightower/envconfig v1.4.0/go.mod h1:cccZRl6mQpaq41TPp5QxidR+Sa3axMbJDNb//FQX6Gg=
+github.com/keptn/go-utils v0.6.3-0.20200923070433-95a8b5159363 h1:p+33Y6OoBjdbEyeEYZZwINN4IO7SGZH71uNsEDTlN2I=
```

Comment:
what's going on here, why are we adding go-utils in versino 0.6.3?

Reply:
this is the latest commit ID. Not sure why it starts with 0.6.3, but this is generated by go modules when targeting a specific commit